styles/diamond-ui.css

```diff
@@ -1,3 +1,5 @@
 @import url('../node_modules/modern-normalize/modern-normalize.css');
 @import url('./tokens.css');
 @import url('./base.css');
+
+@import url('../components/composition/Wrap/Wrap.css');
```

Comment:
We could add https://www.npmjs.com/package/postcss-import-ext-glob to pull these in, doesn't necessarily need to be part of this PR though.

Reply:
Added a ticket for this one